packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -729,6 +729,37 @@ export function isSuspenseInstancePending(instance: SuspenseInstance) {
 export function isSuspenseInstanceFallback(instance: SuspenseInstance) {
   return instance.data === SUSPENSE_FALLBACK_START_DATA;
 }
+export function getSuspenseInstanceFallbackErrorDetails(
+  instance: SuspenseInstance,
+) {
+  const nextSibling = instance.nextSibling;
+  let errorMessage, errorComponentStack, errorHash;
+  if (
+    nextSibling &&
+    nextSibling.nodeType === ELEMENT_NODE &&
+    nextSibling.nodeName.toLowerCase() === 'template'
+  ) {
+    const hash = ((nextSibling: any): HTMLTemplateElement).dataset.hash;
+    const msg = ((nextSibling: any): HTMLTemplateElement).dataset.msg;
+    const stack = ((nextSibling: any): HTMLTemplateElement).dataset.stack;
+    if (hash !== null) errorHash = hash;
+    if (msg !== null) errorMessage = msg;
+    if (stack !== null) errorComponentStack = stack;
+  }
+  return {errorMessage, errorComponentStack, errorHash};
+}
+export function getSuspenseInstanceFallbackErrorHash(
```

Comment:
This function is no longer used. Should it be used in DEV? The only reason the other one should be used in prod is because it might contain an error message but only for the one case that maybe should be a hash anyway.

Reply:
Hmm I actually removed it but github is showing it in the diff for some reason... have you see that before? Makes it hard to trust diffs...